### PR TITLE
Rborder/layouts

### PR DESCRIPTION
**Pull request by Robert-Border**
### Changelog
- Update application.html.erb layout to import additional scss files
- Modify lectures to render as Cards
- Fix conflicts with home.scss file and main application.scss classes
- Add font-awesome gem and configure links to renders as icons

### Manual testing
- Tested locally (Render in course_sessions show without error)
- rake assets:precompile runs successfully without error
image

### Checklist:
- [x]  My code follows the style guideline
- [x]  I have performed a self-review of my own code
- [ ]  I have made corresponding changes to the documentation
- [ ]  Add tests if required

UI Screenshots:
image

image

### CODE REVIEW

File: app/views/course_sessions/show.html.erb [hunk 1]
```diff
@@ -1,73 +1,34 @@
...
-      <th>Homework</th>
-      <th>Teacher</th>
-      <th>CarrotU Session</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
+<p id="notice"><%= notice %></p>
+  <div class="container mx-auto">
```
Comment (shaolinhoward) [inline app/views/course_sessions/show.html.erb:9]: nit: I think this was supposed to be at the same indentation as the notice block

File: app/views/course_sessions/show.html.erb [hunk 2]
```diff
@@ -1,73 +1,34 @@
...
-        <% else %>
-          <td><%= link_to 'Add Homework', new_homework_path(lecture.id) %></td>
-        <% end %>
-        <td><%= lecture.teacher_id %></td>
-        <td><%= lecture.course_session_id %></td>
-        <td><%= link_to 'Show', lecture %></td>
-        <td><%= link_to 'Edit', edit_lecture_path(lecture) %></td>
-        <td><%= link_to 'Destroy', lecture, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
+      <div class="card bg-white mb-3" style="max-width: 100rem;">
```
Comment (shaolinhoward) [inline app/views/course_sessions/show.html.erb:11]: yay! begone table layout!

File: app/views/course_sessions/show.html.erb (hunk 1 above)
Comment (Robert-Border) [reply app/views/course_sessions/show.html.erb]: Fixed indentation.

File: Gemfile [hunk 3]
```diff
@@ -35,6 +35,9 @@ gem 'bootsnap', '>= 1.4.2', require: false
 gem 'bootstrap', '~> 4.5.0'
 gem 'jquery-rails'
 
+# Font Awesome for icons
+gem 'font-awesome-rails'
```
Comment (muffy) [inline Gemfile:39]: It really is awesome 😄

**Pull request merged**